app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate params[:session][:password]
```

Comment:
Toán tử `&.` có tác dụng gì

Reply:
giúp code ngắn gọn hơn, debug dễ dàng hơn và sẽ đưa ra exception nếu 1 phương thức không tồn tại được gọi